### PR TITLE
Fix button artifacts on the main toolbar and effects panel

### PR DESCRIPTION
**Pull request by kryksyh**
Resolves: https://github.com/audacity/audacity/issues/6892

### Main toolbar
image

### Main toolbar
image
image

### Effects panel
image

image

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: src/widgets/AButton.cpp [hunk 1]
```diff
@@ -173,6 +174,7 @@ void AButton::SetFrameMid(int mid)
 void AButton::Init(wxWindow* parent, wxWindowID id, const wxPoint& pos, const wxSize& size, bool toggle)
 {
    SetBackgroundStyle(wxBG_STYLE_PAINT);
+   SetBackgroundColour(theTheme.Colour(clrMedium));
```
Comment (vsverchinsky) [inline src/widgets/AButton.cpp:177]: There are few places that may require backgroud colour override as they use different one: `RealtimeEffectsPanel.cpp` (opened via Effects button on track panel, then add effect) and `EffectUI.cpp`(open effect ui by clicking on the button with effect name)
Did you check them?

**Review by vsverchinsky — APPROVED** [review]

File: src/widgets/AButton.cpp (hunk 1 above)
Comment (kryksyh) [reply src/widgets/AButton.cpp:177]: >Did you check them?
I have not, I need to fix those, thanks!
image

Comment (kryksyh) [reply src/widgets/AButton.cpp:177]: @vsverchinsky  Should look better now, please check

Comment (vsverchinsky) [reply src/widgets/AButton.cpp:177]: изображение
Click an effect name to open effect UI. I get this under Windows platform

Comment (kryksyh) [reply src/widgets/AButton.cpp:177]: @vsverchinsky right, missed that one with the dark theme enabled. Should be fixed now.

Why do effect dialogs not use the selected theme though?

Comment (vsverchinsky) [reply src/widgets/AButton.cpp:177]: 👍 
Don't know for sure, but probably it was something that was missed during redesign or there might have been some problems with that

**Review by vsverchinsky — APPROVED** [review]

**Comment by dozzzzer** [comment]
Tested. I no longer see the original issue:

image

**Comment by CennoxX** [comment]
I also tested the fix and can confirm the issue is solved.

**Pull request merged**